src/server/app.js

```diff
@@ -26,4 +26,4 @@ export default compose([
     actionDispatcher(),
     router(),
     renderer(),
-]);
+].filter(Boolean));
```

Comment:
If you use this syntax, it would be better to have 
```js
process.env.NODE_ENV !== 'production' && 
    function* setCORS(next) { this.set('Access-Control-Allow-Origin', '*'); yield next; },
``` 
instead of the previous syntax few lines before